src/main/java/com/epam/auto/test/level2/Task_2_4_WebDriver_BringItOn.java

```diff
@@ -0,0 +1,17 @@
+package com.epam.auto.test.level2;
+
+import com.epam.auto.test.level2.pages.PastebinMainPage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import java.util.concurrent.TimeUnit;
+
+public class Task_2_4_WebDriver_BringItOn {
+
+    public static void main(String[] args) throws InterruptedException {
+        WebDriver webDriver = new ChromeDriver();
+        new PastebinMainPage(webDriver).createNewPaste();
+        TimeUnit.SECONDS.sleep(5);
```

Comment:
This is a very bad approach. Please use waiters in an appropriate way and don't use Thread.sleep and etc.